docs/source/pages/get_started/index.md

```diff
@@ -0,0 +1,71 @@
+:html_theme.sidebar_secondary.remove:
+
+# Get Started
+
+See [Set up a project](set-up-a-project_) to set up `datashuttle` to manage a new or existing
+project for folder creation, validation and transfer.
+
+Alternatively, to quickly validate an existing [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/latest/index.html)
+project without setting it up as a managed project, see [Validate an existing project](quick-validate-projects).
```

Comment:
I think these parts shouldn't be necessary if the cards were described fully

Reply:
Agree, will discuss on other comment